src/apis/youves/index.ts

```diff
@@ -1,7 +1,8 @@
+import { contracts } from '@temple-wallet/youves-sdk/dist/networks';
+import { AssetDefinition } from '@temple-wallet/youves-sdk/dist/networks.base';
+import { UnifiedStakeExtendedItem } from '@temple-wallet/youves-sdk/dist/staking/unified-staking';
```

Comment:
These imports became too inconvenient because there was no `src/index.ts` file in the original module. We should add it to our fork, make a new release, and use it here.